src/app/transformers/jade-bootstrap3.transformer.ts

```diff
@@ -0,0 +1,64 @@
+import {Transformer} from "./transformer";
+
+export class JadeBootstrap3Transformer implements Transformer {
+  name = 'Jade & Bootstrap 3';
+  settings = [{
+    key: 'validations',
+    label: 'add bootstrap 3 validation classes',
+    value: true
+  }];
+
+  getFormHeader(): string {
+    return `
+.row
+  .col-md-12.col-sm-6.col-xs-12
+    .panel.panel-default
+      .panel-heading.clearfix
+        i.icon-calendar
+        h3.panel-title Title
+      .panel-body
```

Comment:
This is a very specific implementation (grid+responsive classes + panels + icons).
I think its better keeping it minimal, so people can do their own adaptation.
Perhaps use [HTML2Jade](http://html2jade.org/) to convert the current BS3+Angular template?

Reply:
I don't use angular so I tried to keep it general, the icons are from the default BS3 and the grid is quite standard for inline forms (2, 10) 
I thought about adding more options to the object but I think it'll complicate staff without adding value
I can add a "Pug (FKA Jade) & Angular & Bootstrap 3" template